modules/variables.tf

```diff
@@ -65,13 +65,6 @@ variable "grafana_admin_password" {
   default     = null
 }
 
-variable "grafana_generic_oauth_extra_args" {
-  description = "override default oauth value(s)"
-  type        = map(string)
-  default = {
-    autoassign_org_role       = "Editor"
-  }
-}
```

Comment:
Why did you remove this?

Reply:
It was my first try ... it does not work and it has nothing to do in a "doc" commit ;-)
My modification is in modules/values.tmpl.yaml